bokeh/sphinxext/bokeh_plot.py

```diff
@@ -268,6 +301,10 @@ def env_merge_info(app, env, docnames, other):
 
 def setup(app):
     """ Required Sphinx extension setup function. """
+    app.add_node(sampledata_node,
+                 html=(visit_sampledate_node, depart_sampledate_node),
+                 latex=(visit_sampledate_node, depart_sampledate_node),
+                 text=(visit_sampledate_node, depart_sampledate_node))
```

Comment:
This lines are needed, because of the Admition part.

Reply:
I'll have to check this out locally in the next few days and play around. I think it can be done but I don't know how offhand.